src/utils/hydrate-config.ts

```diff
@@ -1,7 +1,13 @@
 import R from 'ramda'
 import { substituteWithEnv } from './substitute-with-env'
 
-export const hydrateConfig = (runtimeEnvironment: string) =>
+export type InnerHydrateFunction = (
```

Comment:
the name confuses me a bit, is there a more expressive one maybe? what is this function doing exactly?

Reply:
Open for suggestions.

`hydrateConfig` is curried. It does the same thing as before, which i described in the first PR already (https://github.com/strong-config/node/pull/5#discussion_r328724018). Just with types so it should be clearer than before.